broker/aws/impl.go

```diff
@@ -392,7 +397,14 @@ func (b *Broker) getConsoleURLForAccountRole(accountName string, roleName string
 	creds := url.QueryEscape(string(bcreds[:]))
 	b.logger.Debugf(1, "sessionCredentials-escaped=%v", creds)
 
-	req, err := http.NewRequest("GET", "https://signin.aws.amazon.com/federation", nil)
+	federationUrl := "https://signin.aws.amazon.com/federation"
+	awsDestinationURL := "https://console.aws.amazon.com/"
+	if region == "us-gov-west-1" {
```

Comment:
Please change this test to: `if strings.HasPrefix(region, "us-gov-")` so that other GovCloud regions are supported.